.pipelines/v2/templates/job-build-project.yml

```diff
@@ -414,9 +457,10 @@ jobs:
       & '.pipelines/verifyPossibleAssetConflicts.ps1' -targetDir '$(build.sourcesdirectory)\$(BuildPlatform)\$(BuildConfiguration)\WinUI3Apps'
     displayName: Audit WinAppSDK applications path asset conflicts
 
-  - pwsh: |-
-      & '.pipelines/verifyNoticeMdAgainstNugetPackages.ps1' -path '$(build.sourcesdirectory)\'
-    displayName: Verify NOTICE.md and NuGet packages match
+  - ${{ if eq(parameters.useLatestWinAppSDK, false) }}:
```

Comment:
can you add a comment explaining why `useLatestWinAppSDK` needs `NOTICE.md` not to be verified

Reply:
Done.